scripts/start_docker_bblfsh.sh

```diff
@@ -1,27 +0,0 @@
-#!/bin/bash
```

Comment:
I find those 3 to be useful examples of how to run containers with all the options, etc individually with just `docker` that i.e may be useful for local debugging.

Do you think we could keep them around and delete, when this knowledge get's transferred to the  a Developer's docs? 

It's nice to have `docker-compose` to run all of them and I know that all the options, etc for individual docker command can be recovered from compose configuration, but I still see value for having those individual commands in some place.

Reply:
don't see any value to keep scripts as documentation. but whatever.